crates/cli_utils/src/helpers.rs

```diff
@@ -478,3 +483,12 @@ pub fn known_bad_file(file_name: &str) -> PathBuf {
 
     path
 }
+
+/// Copies the glue source files for zig platforms from the roc builtins
+/// this is only temporary, see comments in crates/copy_zig_glue/src/main.rs
+fn initialize_zig_test_platforms() {
```

Comment:
It may not be temporary actually -- perhaps we should update this comment?

We will want to keep test platforms, and zig hosts are very simple.